### DISCUSSION TITLE
Feature: SSH list

### DISCUSSION THREAD

**Discussion by antmordel** [PatrickF1/fzf.fish#96] [Archive] [answered]
Hi Patrick. I'll love to have an ssh list of hosts (retrieved from known_hosts) to do ssh to. If you need help I can try to implement it.

**Comment by PatrickF1** [comment, marked as answer]
Hi @antmordel thanks for the idea. Ugh, this is a tough one. To be honest I'm on the fence for two reason
1) doesn't seem like more than 1/5 of current users would use it and I don't want fzf.fish to be bloated. However, I do see that this would be very useful for those that do.
2) I'm not very familiar with the syntax/format of `known_hosts` but it does seem hard to parse. It seems very difficult to write something that can correctly parse and extract hosts 100% of the time. I'm sure you know more about it than me. Can you tell me about how you'd parse it?

**Comment by antmordel** [comment]
That's true. I have all my ssh hosts unhashed. But I have read there are cases that the hosts are in hashed manner. We can close this issue then. Thanks Patrick.
